Tests/appstoreconnect-cliTests/Operations/CreateCertificateOperationTests.swift

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+@testable import AppStoreConnectCLI
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+import XCTest
+
+private enum TestError: LocalizedError {
+    case createError
+
+    var localizedDescription: String {
+        return "Error when create certificate"
+    }
+}
+
+final class CreateCertificateOperationTests: XCTestCase {
+    typealias Dependencies = CreateCertificateOperation.Dependencies
+
+    let options = CreateCertificateOptions(
+        certificateType: .iOSDevelopment,
+        csrContent: "")
+
+    func testCreateCertificate_success() {
```

Comment:
🤔 when the test outputs the result we get a little bit of redundancy here:
e.g 
`Test Case '-[appstoreconnect_cliTests.CreateCertificateOperationTests testCreateCertificate_success]' passed (0.099 seconds).`

Maybe we should shorten the prefixes to just the function we are testing:
```suggestion
    func testExecute_success() {
```